zipline/pipeline/loaders/blaze/events.py

```diff
@@ -63,8 +64,7 @@ def __init__(self,
                  resources=None,
                  odo_kwargs=None,
                  data_query_time=None,
-                 data_query_tz=None,
-                 dataset=None):
```

Comment:
this still needs to be an argument so that downstream consumers may change it. We just want to use the class level attribute as the default.